src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/UpdateActionPlanDTO.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+data class UpdateActionPlanDTO(
+  val numberOfSessions: Int?,
```

Comment:
ah i see why this was removed from the validation logic. it's optional to allow updates to just include new activities.

can we change the validation logic then to say "if numberOfSessions is not already set, and the request does not contain number of sessions, then error"? or is that handled elsewhere in the 'create action plan' stuf?

Reply:
This can only be done when "submitting the action plan" because up until that point the plan remains incomplete and can be added too incrementally. 
At some point we may have a number of "rules" to ensure the validity of an action plan at the point of submission, e.g. an activity must not exist for an outcome that is not part of the referral. When we build this type of logic we can add the rule for ensuring a number of session exists.